src/clj/game/cards/resources.clj

```diff
@@ -858,6 +859,31 @@
                  :effect (effect (gain-bad-publicity :corp 1)
                                  (trash card {:cause :ability-cost}))}]}
 
+   "Jackpot!"
+   (let [jackpot {:interactive (req true)
+                  :delayed-completion true
+                  :req (req (= :runner (:as-agenda-side target)))
+                  :effect (req (show-wait-prompt state :corp "Runner to use Jackpot!")
+                               (continue-ability
+                                 state side
+                                 {:optional
+                                  {:prompt "Trash Jackpot!?"
+                                   :no-ability {:effect (effect (clear-wait-prompt :corp)
+                                                                (effect-completed eid))}
+                                   :yes-ability
+                                   {:prompt "Choose how many [Credit] to take"
+                                    :choices {:number (req (get-in card [:counter :credit] 0))}
+                                    :delayed-completion true
+                                    :effect (req (gain state :runner :credit target)
+                                                 (system-msg state :runner (str "trashes Jackpot! to gain " target " credits"))
+                                                 (clear-wait-prompt state :corp)
+                                                 (trash state :runner eid card {:cause :ability-cost}))}}}
```

Comment:
This effect is not the cost of a triggered ability; the trash is a side effect. `:ability-cost` is for "[Trash]: ..." abilities, notably to trigger Geist / Tech Trader.